generators/service-push/templates/swift/instrumentation.swift

```diff
@@ -1,9 +1,9 @@
 import LoggerAPI
+import CloudEnvironment
 import BluemixPushNotifications
 
-var pushNotifications: PushNotifications!
-
-func initializeServicePush() throws {
+func initializeServicePush(cloudEnv: CloudEnv) throws -> PushNotifications {
+    let pushNotifications: PushNotifications
```

Comment:
If you move this declaration below the guard you won't need the type annotation since it would be inferred by the assignment.

Reply:
done